packages/labs/src/components/popover/popover2.tsx

```diff
@@ -288,8 +289,9 @@ export class Popover2 extends AbstractComponent<IPopover2Props, IPopover2State>
         );
 
         const children = this.understandChildren();
-        const targetTabIndex = this.props.openOnTargetFocus && this.isHoverInteractionKind() ? 0 : undefined;
+        const targetTabIndex = this.props.openOnTargetFocus && isHoverInteractionKind ? 0 : undefined;
         const target: JSX.Element = React.cloneElement(children.target, {
+            className: classNames({ [Classes.ACTIVE]: isOpen && !isHoverInteractionKind }),
```

Comment:
must add `children.target.props.className` to avoid clobbering existing classes.